common/src/main/java/com/wynntils/mc/render/FontRenderer.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.mc.render;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.mc.mixin.accessors.MinecraftAccessor;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.utils.objects.CustomColor;
+import java.util.Arrays;
+import java.util.List;
+import net.minecraft.client.gui.Font;
+
+public class FontRenderer {
+    private static final FontRenderer INSTANCE = new FontRenderer();
+    private final Font font;
+
+    private static final int NEWLINE_OFFSET = 10;
+
+    public FontRenderer() {
+        this.font = ((MinecraftAccessor) McUtils.mc()).getFont();
+    }
+
+    public static FontRenderer getInstance() {
+        return INSTANCE;
+    }
+
+    public int drawLine(
```

Comment:
Maybe "renderText"? "drawLine" sounds like it will draw a straight line on the screen.

Reply:
Fixed.